python/its-info/its_info/main.py

```diff
@@ -0,0 +1,132 @@
+# Software Name: its-info
+# SPDX-FileCopyrightText: Copyright (c) 2022 Orange
+# SPDX-License-Identifier: MIT
+# Author: Yann E. MORIN <yann.morin@orange.com>
+
+import argparse
+import configparser
+import json
+import linuxfd
+import netifaces
+import os
+import paho.mqtt.client
+import time
+
+
+def main():
+    try:
+        info = MQTTInfoClient()
+        info.loop_forever()
+    except Exception as e:
+        print(e)
+        os._exit(1)
```

Comment:
why not use `sys.exit(n)` ? Are you into a subprocess for?

Reply:
[`sys.exit()`](https://docs.python.org/3/library/sys.html#sys.exit) merely raises _a [SystemExit](https://docs.python.org/3/library/exceptions.html#SystemExit) exception, signaling an intention to exit the interpreter_.

Notably, if there are other threads still running, there are cases where that will
not actually terminate the program. On the other hand, `os._exit()` will always
successfully terminate.

In our case, we properly catch Ctrl-C and any similar "nomal" interruptions to do a
clean shutdown. But if we receive any unexpected exception, we do not want to linger
around forever; since we had an unexpected exception, we are in an unpredicatble
state, so we don't have much cleanup to do, so calling `os._exit()` is the best we
can do.

Reply:
Okay. I think it would be great to put your response as a comment into the code so. we're using a (python semantic) private method, preffixed by "_": let's justify why we considere we're allowed ;).